include/dbus_utility.hpp

```diff
@@ -210,6 +210,23 @@ inline void getAssociationEndPoints(
         });
 }
 
+inline void getAssociatedSubTree(
+    const sdbusplus::message::object_path& associatedPath,
+    const sdbusplus::message::object_path& path, int32_t depth,
+    std::span<const std::string_view> interfaces,
+    std::function<void(const boost::system::error_code&,
+                       const MapperGetSubTreeResponse&)>&& callback)
+{
+    crow::connections::systemBus->async_method_call(
```

Comment:
Please use dbus::utility::getAssociatedSubTree instead of relying on crow::connections::systemBus->async_method_call.

Reply:
> Please use dbus::utility::getAssociatedSubTree instead of relying on crow::connections::systemBus->async_method_call.

This is just a cherry-pick from upstream to have `getAssociatedSubTree()` in BMCWEB - which was missed in previous cherry-picking of 

```
commit f99586afcbf03f8b2fe1c0ace84b8c52f332b0f9
Author: Lakshmi Yadlapati <lakshmiy@us.ibm.com>
Date:   Tue Jun 20 23:46:10 2023 -0500

    dbus_utility: Support new ObjectMapper methods
...
    https://gerrit.openbmc.org/c/openbmc/bmcweb/+/62522
```

In other words, this is not a case of using the function, but it is to add the function in bmcweb.